hawkular-bus-common/src/main/java/org/hawkular/bus/common/MessageProcessor.java

```diff
@@ -338,42 +339,6 @@ public MessageId sendWithBinaryData(ProducerConnectionContext context, BasicMess
     }
 
     /**
-     * Same as {@link #sendRPC(ProducerConnectionContext, BasicMessage, Class, Map)} with <code>null</code> headers.
-     */
-    public <R extends BasicMessage> ListenableFuture<BasicMessageWithExtraData<R>> sendRPC(
```

Comment:
I removed that method because I did not see it used anywhere in any of the of the component repos. I only found it used in a one of the samples and in a test method.